apps/asap-server/src/controllers/research-outputs.ts

```diff
@@ -120,6 +135,48 @@ export default class ResearchOutputs implements ResearchOutputController {
       ),
     };
   }
+  async create(
+    researchOutputData: ResearchOutput,
+    teamId: string,
+  ): Promise<Partial<ResearchOutputResponse>> {
+    const { id: researchOutputId } = await this.createResearchOutput(
```

Comment:
is there any reason why you extracted these methods to separate, private methods? its not like we are reusing them and not like we would want to test private methods either. It's not a huge deal but im usually against anything that we just dont need right now (YAGNE all the way)

Reply:
unless its just for visibility - so like - when you open the method you know it creates the RO then associates the team, but i dont know..

Reply:
It's just for visibility. 

We could have had them as functions inside the main function, but then they need to be declared first. If they are standalone function we need to pass in the client.